dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/notification/ProgramNotificationTemplate.java

```diff
@@ -68,6 +68,8 @@
 
     private Boolean notifyParentOrganisationUnitOnly;
 
+    private boolean sendRepeatable;
```

Comment:
Don't we need a Flyway migration here as `ProgramNotificationTemplate` is a persisted class? For primitive boolean we need to first add the field, populate with false values where null, then set to not-null.

Reply:
yes, @larshelge flyway is needed.
We need to add the field and set it to false where null. By default, it's false for new templates.